style/ember/README.md

```diff
@@ -2,3 +2,13 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+
+Testing
+-------
+
+* Visually separate the setup, exercise and assertion phases by:
```

Comment:
Maybe remove this part since that's (I think) already a best practice and unnest the two rules below?